### PR TITLE
windows install: save script output

### PR DESCRIPTION
**Pull request by Thykof**

### CODE REVIEW

File: installer/windows/scripts/configure_network_interfaces.bat [hunk 1]
```diff
@@ -7,16 +7,31 @@ SETLOCAL ENABLEDELAYEDEXPANSION
...
 
-    echo Configuring !networkAdapterName: =!...
+    call :WriteToLog "Configuring !networkAdapterName: =!..."
 
     NETSH interface ipv4 set dnsservers "!networkAdapterName: =!" static 127.0.0.1 primary
     NETSH interface ipv6 set dnsservers "!networkAdapterName: =!" static ::1 primary
     if %errorlevel% NEQ 0 (
-        ECHO "Failed to configure !networkAdapterName: =!"
+        call :WriteToLog "Failed to configure !networkAdapterName: =!"
+        pause
```
Comment (Thykof) [inline installer/windows/scripts/configure_network_interfaces.bat:16]:
```suggestion
```

File: installer/windows/scripts/configure_acrylic.bat [hunk 2]
```diff
@@ -31,10 +31,25 @@ ECHO ::1 *.massa >> "%acrylic_config%"
 NET STOP "AcrylicDNSProxySvc"
 NET START "AcrylicDNSProxySvc"
 if %errorlevel% NEQ 0 (
-    ECHO "Failed to restart Acrylic DNS Proxy Service"
+    call :WriteToLog "Failed to restart Acrylic DNS Proxy Service"
+    pause
```
Comment (Thykof) [inline installer/windows/scripts/configure_acrylic.bat:35]:
```suggestion
```

File: installer/windows/scripts/configure_acrylic.bat [hunk 3]
```diff
@@ -31,10 +31,25 @@ ECHO ::1 *.massa >> "%acrylic_config%"
...
     EXIT 1
 )
 
 ENDLOCAL
 
+call :WriteToLog "Success"
+pause
```
Comment (Thykof) [inline installer/windows/scripts/configure_acrylic.bat:42]:
```suggestion
```

File: installer/windows/scripts/configure_network_interfaces.bat [hunk 4]
```diff
@@ -7,16 +7,31 @@ SETLOCAL ENABLEDELAYEDEXPANSION
...
         EXIT 1
     )
 )
 
 ENDLOCAL
 
+call :WriteToLog "Success"
+pause
```
Comment (Thykof) [inline installer/windows/scripts/configure_network_interfaces.bat:24]:
```suggestion
```

File: installer/windows/scripts/configure_acrylic.bat [hunk 5]
```diff
@@ -31,10 +31,23 @@ ECHO ::1 *.massa >> "%acrylic_config%"
...
+    call :WriteToLog "Failed to restart Acrylic DNS Proxy Service"
     EXIT 1
 )
 
 ENDLOCAL
 
+call :WriteToLog "Success"
 EXIT 0
+
+:: decalre a function to log and print a message
```
Comment (gregLibert) [inline installer/windows/scripts/configure_acrylic.bat:43]:
```suggestion
:: declare a function to log and print a message
```

**Review by gregLibert — APPROVED** [review]

**Comment by thomas-senechal** [comment]
Shouldn't we use only one way of logging output of our scrpits ? 
I would go with the way that logs EVERYTHING of the script, it will give us way more information about what's happening

Also, I don't think having a file for each of our scripts is a good idea, because in case of error, that means that we will ask users for 4 files ? 5 if we count for the `MSI{...}.log` file 

I think we should have a single file such as `massastation_install.log`, and at the beginning of each script, we could log the name of the script that is starting to make it easier for us to know what is being logged and what is failing or not

File: installer/windows/scripts/configure_acrylic.bat [hunk 6]
```diff
@@ -31,10 +38,12 @@ ECHO ::1 *.massa >> "%acrylic_config%"
 NET STOP "AcrylicDNSProxySvc"
 NET START "AcrylicDNSProxySvc"
 if %errorlevel% NEQ 0 (
-    ECHO "Failed to restart Acrylic DNS Proxy Service"
+    echo "Failed to restart Acrylic DNS Proxy Service"
```
Comment (Thykof) [inline installer/windows/scripts/configure_acrylic.bat:41]:
```suggestion
    ECHO "Failed to restart Acrylic DNS Proxy Service"
```

File: installer/windows/scripts/configure_acrylic.bat [hunk 7]
```diff
@@ -17,8 +24,8 @@ if not exist "%acrylic_config%" (
 :: Check if the TLD is already in the file
 FINDSTR /c:".massa" "%acrylic_config%" >nul 2>&1
 if %errorlevel%==0 (
-    ECHO TLD already in the file
-    goto :EOF
+    echo "TLD already in the file"
```
Comment (Thykof) [inline installer/windows/scripts/configure_acrylic.bat:27]:
```suggestion
    ECHO "TLD already in the file"
```

File: installer/windows/scripts/configure_network_interfaces.bat [hunk 8]
```diff
@@ -1,22 +1,29 @@
...
     SET "networkAdapterName=%%B"
 
-    echo Configuring !networkAdapterName: =!...
+    echo "Configuring !networkAdapterName: =!..."
 
     NETSH interface ipv4 set dnsservers "!networkAdapterName: =!" static 127.0.0.1 primary
     NETSH interface ipv6 set dnsservers "!networkAdapterName: =!" static ::1 primary
     if %errorlevel% NEQ 0 (
-        ECHO "Failed to configure !networkAdapterName: =!"
+        echo "Failed to configure !networkAdapterName: =!"
```
Comment (Thykof) [inline installer/windows/scripts/configure_network_interfaces.bat:22]:
```suggestion
        ECHO "Failed to configure !networkAdapterName: =!"
```

File: installer/windows/scripts/configure_acrylic.bat [hunk 9]
```diff
@@ -17,8 +24,8 @@ if not exist "%acrylic_config%" (
 :: Check if the TLD is already in the file
 FINDSTR /c:".massa" "%acrylic_config%" >nul 2>&1
 if %errorlevel%==0 (
-    ECHO TLD already in the file
-    goto :EOF
+    ECHO "TLD already in the file"
```
Comment (Thykof) [inline installer/windows/scripts/configure_acrylic.bat:27]:
```suggestion
    ECHO TLD already in the file
```

File: installer/windows/scripts/configure_network_interfaces.bat [hunk 10]
```diff
@@ -1,13 +1,20 @@
...
+
 SETLOCAL ENABLEDELAYEDEXPANSION
 
 :: List available Network Interfaces and set dnsservers to 127.0.0.1 (localhost) and ::1 (IPv6 localhost)
 :: This is required for Acrylic to work properly
 for /f "skip=1 delims=" %%A in ('wmic nic where "netenabled=true" get netconnectionID') do @for /f "delims=" %%B in ("%%A") do (
     SET "networkAdapterName=%%B"
 
-    echo Configuring !networkAdapterName: =!...
+    echo "Configuring !networkAdapterName: =!..."
```
Comment (Thykof) [inline installer/windows/scripts/configure_network_interfaces.bat:17]:
```suggestion
    echo Configuring !networkAdapterName: =!...
```

File: installer/windows/scripts/configure_network_interfaces.bat [hunk 11]
```diff
@@ -19,4 +26,4 @@ for /f "skip=1 delims=" %%A in ('wmic nic where "netenabled=true" get netconnect
 
 ENDLOCAL
 
-EXIT 0
+) >> %LOG_FILE% 2>&1
```
Comment (Thykof) [inline installer/windows/scripts/configure_network_interfaces.bat:29]:
```suggestion
EXIT 0

) >> %LOG_FILE% 2>&1
```

File: installer/windows/scripts/configure_acrylic.bat [hunk 12]
```diff
@@ -18,7 +25,7 @@ if not exist "%acrylic_config%" (
 FINDSTR /c:".massa" "%acrylic_config%" >nul 2>&1
 if %errorlevel%==0 (
     ECHO TLD already in the file
-    goto :EOF
+    EXIT 0
```
Comment (Thykof) [inline installer/windows/scripts/configure_acrylic.bat:28]: @thomas-senechal I change this to exit 0, do you think it's a problem?

Comment (thomas-senechal) [reply installer/windows/scripts/configure_acrylic.bat:28]: Looks good to me !

**Pull request merged**